findphotodates.py

```
@@ -0,0 +1,315 @@
+    print("Files found:")
+    for ext, count in sorted(file_counts.items()):
+        print(f"  {ext.upper()}: {count}")
+
+    # Calculate date summaries
+    date_counter = Counter()
+    for _, date, _ in photo_data:
+        if date:
+            try:
+                day = date.split(" ")[0]  # Extract just the date portion
+                date_counter[day] += 1
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)